packages/espresso/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
This can be removed; tests are dev-only, so are exempt. You just need to add the label to override changelog detection.